src/tests/test_helpers.c

```diff
@@ -13,22 +13,35 @@ int reverse_bits_int(int in) {
     return res;
 }
 
+s21_decimal convert_gmp_to_decimal(mpz_t input) {
+    s21_decimal res = {0};
+    mpz_export(res.bits, 0, 1, sizeof(res.bits[0]), -1, 0, input);
+    return res;
+}
+
+int get_rand(int min, int max) {
+    int val = (int)rand() % 2;
```

Comment:
I think, we should insert setting random seed inside this function. The problem is that in CK_FORK mode the seed is set only once & we are getting the same seed for all forks.